podvm/Makefile

```diff
@@ -37,8 +37,8 @@ ifeq ($(PODVM_DISTRO),ubuntu)
 	-var cloud_image_url=${UBUNTU_IMAGE_URL} \
 	-var cloud_image_checksum=${UBUNTU_IMAGE_CHECKSUM} \
 	-var se_boot=${SE_BOOT})
-ifeq ($(SE_BOOT),1)
-	$(eval PACKER_DEFAULT_OPTS = -var boot_wait=480s )
+ifeq ($(ARCH),s390x)
```

Comment:
My understanding is to increase the timeout to 10 minutes both for SE and non-SE image for s390x? Previous timeout for non-SE image was 5 minutes and SE image was 8 minutes?
Is it good enough to increase SE image from 8 minutes to 10 minutes and non-SE image from 5 minutes to 10 minutes? Shall we increase timeout more larger?

Reply:
Given the errors like below `errored after 7 minutes 13 seconds`, looks 10 minutes is not safe?
```
Build 'qemu.ubuntu' errored after 7 minutes 13 seconds: Packer experienced an authentication error when trying to connect via SSH. This can happen if your username/password are wrong. You may want to double-check your credentials as part of your debugging process. original error: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none], no supported methods remain
```

Reply:
> My understanding is to increase the timeout to 10 minutes both for SE and non-SE image for s390x? Previous timeout for non-SE image was 5 minutes and SE image was 8 minutes? Is it good enough to increase SE image from 8 minutes to 10 minutes and non-SE image from 5 minutes to 10 minutes? Shall we increase timeout more larger?

yeah, it's both se and noe-se image, I go through most of the failed job 10 minutes for boot the qemu s390x instance should be enough.